examples/plot_structure_basic/plot_inferred_structure.py

```diff
@@ -0,0 +1,223 @@
+"""
+Plot an inferred 3-d structure with matplotlib
+==============================================
+
+This example showcases how to plot an inferred 3-d structure output by
+PASTIS using matplotlib.
+"""
+
+import numpy as np
+import itertools
+from mpl_toolkits.mplot3d import axes3d
+import matplotlib
+import matplotlib.pyplot as plt
+import pandas as pd
+from scipy import interpolate
+
+##############################################################################
+# Get a dataframe for the inferred structure's coordinates.
+df = pd.read_csv('data/struct_inferred.000.coords', delimiter=" ", header=None)
+
+
+
+##############################################################################
+# Define the interpolation function.
+def interpolate_chromosomes(X, lengths, eps=1e-1, smooth=0):
+    
+    # Return a smoothed interpolation of the chromosomes
```

Comment:
These should not be comments but docstrings. If you leave those as docstrings, does that causes problems with sphinx-gallery?

Reply:
I think I fixxed it.